### PR TITLE
E2e landing page

### PR DESCRIPTION
**Pull request by shewood**
Added a cypress test to the project
added docker-compose 

you will likely need to run npm i first

### CODE REVIEW

File: cypress.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# by setting CI environment variable we switch the Cypress install messages
+# to small "started / finished" and avoid 1000s of lines of progress messages
+ENV CI=1
+RUN npm install --save-dev cypress-axe axe-core
+
+# verify that cypress has been installed correctly
+RUN npx cypress verify
+
+COPY cypress ./cypress
+COPY cypress.json .
```
Comment (EricDube) [inline cypress.Dockerfile]: We should build this and store it in our repo so we can reuse it, this looks like an image that shouldn't change often

Comment (P0NDER0SA) [reply cypress.Dockerfile]: @EricDube let's discuss this for sure

**Comment by P0NDER0SA** [comment]
@shewood This appears to be a good start.  Following along with @EricDube idea, let's talk about specifics on the containerization of this process. i'll need to move the docker compose file and centralize some work, but we'll need to talk about it so we are both on the same page.

File: cypress.Dockerfile (hunk 1 above)
Comment (shewood) [reply cypress.Dockerfile]: Agreed.

**Pull request merged**